assets/js/script.js

```diff
@@ -0,0 +1,353 @@
+
+let startButton = document.getElementById("startBtn");
+let questionContainerEl = document.getElementById("questionCard");
+let questionEl = document.getElementById("question");
+let answerBtnsEl = document.getElementById("answerBtns");
+var currentQuestion = 0;
+let score = 0;
+let currentScoreEl = document.getElementById('currentScoreSpan');
+let endGameEl = document.querySelector('.endGame');
+let viewScoresButton = document.getElementById('scoreViewer');
+let submitScoreButton = document.getElementById('scoreSaver');
+let firstContainerEl = document.querySelector('.container');
+let formEl = document.querySelector('.input-group');
+let playAgainBtnEl = document.getElementById('playAgainButton');
+let endScoreEl = document.getElementById('endScoreSpan');
+//Sound Elements
+let dingEl = document.getElementById('correctAudio')
+let corkEl = document.getElementById('corkAudio')
+let songEl = document.getElementById('song')
+let clackEl = document.getElementById('clack')
+
+let timeLeft = 185;
+let answerBtnIndividual = document.getElementById('btn1', 'btn2', 'btn3', 'btn4')
+
+
+//Saving variables
+let scoreName = document.getElementById('nameInput').value;
+let orderedListEL = document.getElementById('highScoreList');
+// let scoreGroup = JSON.parse(localStorage.getItem('highScores'))||[]
+let scoreGroup=[]
+// let orderedListEL = JSON.parse(localStorage.getItem(orderedListEL));
+
```

Comment:
Remove _**all instances**_ of more than one line break. Use detailed comments to separate code if needed.